### PR TITLE
Line indent text annotation app

### PR DESCRIPTION
**Pull request by LineIndent**
This app base foundation - user's create a category (label) and they can highlight a text segment. We can add more details

### CODE REVIEW

File: text_annotation/app2/components/annotation.py
Comment (Alek99) [inline text_annotation/app2/components/annotation.py:1]: For this lets stick to radix components except the accordion that can be chakra

**Review by Alek99 — CHANGES_REQUESTED** [review]

**Comment by Alek99** [comment]
The rx.upload here should use the accept prop to filter out only valid file types, I tried to pass in a txt file but got an error.

**Review by picklelo — COMMENTED** [review]
This is working nicely for me! I'm thinking we should move this to the reflex-examples repo as it may fit better as an example app than a starter template.

**Comment by LineIndent** [comment]
Sounds good - let me know what you guys decide so I can make a new PR in the reflex-examples repo.

**Comment by picklelo** [comment]
@LineIndent I think go ahead and add this in the examples repo and we can merge it

**Comment by LineIndent** [comment]
Great 👍🏼 I made the PR and I'll close this one

**Pull request closed without merging**